components/Tile.tsx

```diff
@@ -1,6 +1,29 @@
-import { Pane, Badge } from 'evergreen-ui';
+import {
+    Pane, Badge, Button, InfoSignIcon, ArrowTopRightIcon,
+} from 'evergreen-ui';
 
 export default function Tile({ tile }) {
+    // assign colors to badges
+    let col = '';
+    if (tile.type === 'Student Sale') {
```

Comment:
create an object (see how Nav/index.tsx uses the rainbow object) instead of an if statement, this applies to both of them.